includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -122,6 +132,12 @@ public function register_get_data_route() {
 	 * @return array Setup Wizard data
 	 */
 	public function get_data() {
+		$home_data = $this->remote_data_provider->fetch( HOUR_IN_SECONDS );
+		if ( is_wp_error( $home_data ) ) {
+			exit;
```

Comment:
Maybe we could return an error (maybe not exactly the one we have in `$home_data` because it's probably from senseilms.com, but not sure) to the rest API to display a friendly error in the frontend?

Reply:
I honestly don't even remember adding this `exit`! 😆 

I'll change it to be more graceful.